packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -811,6 +812,7 @@ test.serial('bulk operation BULK_GRANULE_REINGEST reingests granule IDs returned
       knexOrTransaction: knex,
     });
 
+    callArgs[0].granule.files = [];
```

Comment:
Hm I'm having trouble understanding why this change is necessary. Do you mind explaining?

Reply:
I believe `callArgs[0].granule.file` will already be `[]` 🤔